daprdocs/content/en/developing-applications/building-blocks/observability/app-health.md

```diff
@@ -0,0 +1,73 @@
+| `--app-health-probe-interval` | `dapr.io/app-health-probe-interval` | Number of *seconds* between each health probe | `5` |
+| `--app-health-probe-timeout`  | `dapr.io/app-health-probe-timeout`  | Timeout in *milliseconds* for health probe requests | `500` |
+| `--app-health-threshold`      | `dapr.io/app-health-threshold"`     | Max number of consecutive failures before the app is considered un-healthy | `3` |
+
+> See the [full reference]({{<ref arguments-annotations-overview>}}) for Dapr arguments and annotations for all options and how to enable them.
+
+Additionally, app health checks are impacted by the protocol used for the app channel, which is configured with the `--app-protocol` flag or the `dapr.io/app-protocol` annotation; supported values are `http` (default) or `grpc`.
+
+### Health check paths
+
+When using HTTP for `app-protocol`, Dapr will perform health probes by making a HTTP call to the path specified in `app-health-check-path`, which is `/health` by default.  
+For your app to be considered healthy, the response must have an HTTP status code in the 200-299 range; any other status code is considered a failure. Dapr is only concerned with the status code of the response, and will ignore any response header or body.
+
+When using gRPC for the app channel, Dapr will invoke the method `/dapr.proto.runtime.v1.AppCallbackHealthCheck/HealthCheck` in your application. Most likely, you will use a Dapr SDK to implement the handler for this method.
+
+While responding to a health probe request, your app *may* decide to perform additional internal health checks to determine if it's actually in a state where it's ready to process work from the Dapr runtime. However, this is not required, and it's a choice that depends on your application's needs.
+
+### Intervals, timeouts, and thresholds
+
+When app health checks are enabled, by default Dapr probes your application every 5 seconds; you can configure the interval, in seconds, with `app-health-probe-interval`. These probes happen regularly, regardless of whether your application is healthy or not.
+
+When the Dapr runtime (sidecar) is first started, Dapr waits for a first, successful health probe before considering the app healthy. This means that pub/sub subscriptions, input bindings, and service invocation won't be enabled for your app until a first health check is complete and successful.
+
+Health probe requests are considered successful if the application sends a successful response (as explained above) within the timeout configured in `app-health-probe-timeout`. The default value is 500, corresponding to 500 milliseconds (i.e. half a second).
+
+Before Dapr considers an app to have entered an un-healthy state, it will wait for `app-health-threshold` consecutive failures, whose default value is 3.  
```

Comment:
```suggestion
Before Dapr considers an app to have entered an unhealthy state, it will wait for `app-health-threshold` consecutive failures, whose default value is 3.  
```